configure

```diff
@@ -536,8 +536,8 @@ find_sdlconfig() {
 	IFS="$ac_save_ifs"
 
 	if test -z "$_sdlconfig"; then
-		echo "none found!"
-		exit 1
+		_sdlconfig="pkg-config sdl3"
```

Comment:
Ugh, I did not realize, that this breaks completely systems without pkg-config. Like mine....